lib/ansible/modules/cloud/cloudscale/cloudscale_server.py

```diff
@@ -211,13 +219,19 @@
 flavor:
   description: The flavor that has been used for this server
   returned: success when not state == absent
-  type: str
-  sample: flex-8
+  type: dict
+  sample: { 'slug': 'flex-4', 'name': 'Flex-4', 'vcpu_count': 2, 'memory_gb': 4}
```

Comment:
I would suggest to move this "fix" to a separate PR, and IMHO the same fix would be required for "image" as well.

Reply:
Sure, I reverted 9e7c798 and I'll follow up with another PR that fixes these documentation issues. Thanks!